.github/workflows/ci-unit-tests-os.yml

```
@@ -78,21 +78,15 @@ jobs:
         # TODO: macOS tests were removed for now, see https://github.com/ZcashFoundation/zebra/issues/6824
         os: [ubuntu-latest]
         rust: [stable, beta]
-        features: ["", " --features getblocktemplate-rpcs"]
+        # When vars.EXPERIMENTAL_FEATURES is empty this should become a single matrix item
+        features: ${{ fromJSON(format('["", {0}]', vars.EXPERIMENTAL_FEATURES)) }}
```

Will `features` not miss the `--features` prefix?

---

I think you're right, thanks!

```suggestion
        features: ${{ fromJSON(format('["", "--features {0}"]', vars.EXPERIMENTAL_FEATURES)) }}
```

---

Unfortunately I couldn't fix this because of the limitations of variables and GitHub actions script. See the TODO comment for details.